### DISCUSSION TITLE
Issue flash Milfra TB41 switch

### DISCUSSION THREAD

**Discussion by becksen** [arendst/Tasmota#11261] [Q&A] [answered]
Hi all,

I can't successfully complete flashing Milfra TB41 switch with Tasmota. Getting the following error. I'm using RPI 3B with Raspian to do the flashing. 

Regards:

smarthack-wifi.log:

`dnsmasq: failed to create listening socket for 10.42.42.1: Address already in use
Starting AP on wlan0...
Configuration file: /dev/stdin
wlan0: Could not connect to kernel driver
Using interface wlan0 with hwaddr b8:27:eb:15:02:2d and ssid "vtrust-flash"
wlan0: interface state UNINITIALIZED->ENABLED
wlan0: AP-ENABLED 
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED a0:a4:c5:d8:fc:1b
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-CONNECTED 8c:ce:4e:f1:d5:43
wlan0: AP-STA-DISCONNECTED 8c:ce:4e:f1:d5:43
wlan0: interface state ENABLED->DISABLED
wlan0: AP-STA-DISCONNECTED a0:a4:c5:d8:fc:1b
wlan0: AP-DISABLED 
nl80211: deinit ifname=wlan0 disabled_11b_rates=0
AP closed
Stopping DNSMASQ server...
Restarting NetworkManager...
Failed to restart network-manager.service: Unit network-manager.service not found.`

**Comment by becksen** [comment, marked as answer]
In the processing steps it is repeating those steps:
`SmartConfig complete.
Resending SmartConfig Packets`

**Reply by barbudor** [reply]
Tuya-Convert is not supported here
Please go to the Tuya-Convert Github or DigiBlurDIY
Note that Tuya-Convert can be less and less used as:
- Tuya as massively deployed patches to fix the the hack used to OTA from non Tuya certified servers
- Tuya deploys more and more non-ESP cpu that cannot be flashed with Tasmota

We strongly advise serial-flashing using one of the recommended USB/Serial adapter and Tamotizer software
https://tasmota.github.io/docs/Getting-Started/
